Casks/umts-dev.rb

```diff
@@ -12,9 +12,20 @@
 
   stage_only true
 
+  # version managers
   depends_on formula: 'nodenv'
   depends_on formula: 'rbenv'
 
+  # ruby dependencies
+  depends_on formula: 'libyaml'
+  depends_on formula: 'openssl'
+  depends_on formula: 'readline'
```

Comment:
`rbenv` depends on `ruby-build`, which in turn [depends on](https://formulae.brew.sh/formula/ruby-build) `autoconf`, `libyaml`, `openssl@3`, `pkgconf`, and `readline`.

And also, Ruby 3.3+ [doesn't use readline anymore](https://www.ruby-lang.org/en/news/2023/12/25/ruby-3-3-0-released/).